ansible/roles/screenly/files/screenly_utils.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
 recover(){
-    sudo service screenly-viewer stop
-    rm -rf /home/pi/.screenly /home/pi/screenly_assets
-    tar -xvf "$1" -C /home/pi/
-    sudo service screenly-viewer start
+    service screenly-viewer stop
```

Comment:
Did you try this on a non-resin environment?

Reply:
Not yet, I will try this soon

Reply:
Cool. Let me know as I have a feeling it will break w/out `sudo` on Raspbian.

Reply:
That said - i really don't like being able to run as root.

Reply:
@vpetersson 
Tests were successful.
RaspberryPi 3 B+